account_move_line_purchase_info/migrations/12.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+
+_logger = logging.getLogger(__name__)
+
+__name__ = "Upgrade to 12.0.2.0.0"
+
+
+def update_purchase_id_column(cr):
+    cr.execute("""SELECT column_name
```

Comment:
FYI: if you used openupgrade, this would get simpler:
```suggestion
    if not openupgrade.column_exists(
        cr, 'account_move_line', 'purchase_id'
    ):
```